dotnet/samples/Concepts/Memory/VectorStore_HybridSearch_Simple_AzureAISearch.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Azure;
+using Azure.Identity;
+using Azure.Search.Documents.Indexes;
+using Microsoft.Extensions.VectorData;
+using Microsoft.SemanticKernel.Connectors.AzureAISearch;
+using Microsoft.SemanticKernel.Connectors.AzureOpenAI;
+using Microsoft.SemanticKernel.Embeddings;
+
+namespace Memory;
+
+/// <summary>
+/// A simple example showing how to ingest data into a vector store and then use hybrid search to find related records to a given string and set of keywords.
+///
+/// The example shows the following steps:
+/// 1. Create an embedding generator.
+/// 2. Create an AzureAISearch Vector Store.
+/// 3. Ingest some data into the vector store.
+/// 4. Do a hybrid search on the vector store with various text+keyword and filtering options.
+/// </summary>
+public class VectorStore_HybridSearch_Simple_AzureAISearch(ITestOutputHelper output) : BaseTest(output)
+{
+    [Fact]
+    public async Task ExampleAsync()
```

Comment:
> [!NOTE]
> Consider doing for the other touched samples, following our new Sample Guidelines

No `Async` suffix + more descriptive name.

```suggestion
    public async Task IngestDataAndUseHybridSearch()
```

Reply:
Thanks, fixed.  Will do a separate pass through the others.